.github/workflows/deploy-apis-to-staging.yml

```diff
@@ -481,7 +481,7 @@ jobs:
       - name: Deploy to K8S
         run: |
           cd k8s/gp-model/
-          kubectl apply -f stage-gp-model-api.yaml
+          kubectl apply -f stage-airqo-gp-model-job.yaml
           kubectl rollout restart deployment/airqo-stage-gp-model-job -n staging
```

Comment:
@danielogen does this command work for cronjobs??. I have never seen a cronjob listed as part of deployments

Reply:
Yes @NoahNsimbe, it works just fine.